stories/object-attribute/object-attribute.stories.js

```diff
@@ -0,0 +1,33 @@
+export default {
+    title: 'Dev/Object Attribute',
+    components: ['object-attribute']
+};
+
+export const Text = () => `
+
+<div class="fd-object-attribute" id="standardtext">Standard Object Attribute Text</div>
+`;
+
+Text.parameters = { docs: { disable: true } };
+
+
+export const TextWithLink = () => `
+<div id="firsttext" class="fd-object-attribute">Object Attribute :
+<span id="textwithlink" class="fd-object-attribute--link" role="link" tabindex="0">Text With Link</span></div>
+`;
+
+TextWithLink.parameters = { docs: { disable: true } };
+
+export const DisabledLink = () => `
+    <span id="disabledlink" class="fd-object-attribute fd-object-attribute--link" role="link" tabindex="0" aria-disabled=true>Disabled Object Attribute linked Text</span>
```

Comment:
`aria-disabled` needs quotes around `"true"` (even though it seems to properly resolve in the browser), otherwise this looks fine.